### PR TITLE
Push bookworm binaries for supported platforms

### PR DESCRIPTION
**Pull request by kumaraksh1**
- [x] The purpose of this PR is explained in this message or in an issue. If an issue please include a reference as \#\<issue\_number\>.
- [x] Tests are included and/or updated for code changes.
- [x] Proper license headers are included in each file.

In the following PR, bookworm support was added for python and Php but it was a miss while pushing to prod storage account.
https://github.com/microsoft/Oryx/pull/2337

### CODE REVIEW

File: vsts/scripts/publishSdksFromStagingToProdStorageAccount.sh [hunk 1]
```diff
@@ -108,8 +108,13 @@ function copyPlatformBlobsToProdForDebianFlavor() {
     # Not allowed combinations:
```
Comment (pauld-msft) [inline vsts/scripts/publishSdksFromStagingToProdStorageAccount.sh:108]: could you update this comment in its entirety?

Comment (kumaraksh1) [reply vsts/scripts/publishSdksFromStagingToProdStorageAccount.sh]: Updated, thanks !!

File: vsts/scripts/publishSdksFromStagingToProdStorageAccount.sh [hunk 2]
```diff
@@ -108,8 +108,13 @@ function copyPlatformBlobsToProdForDebianFlavor() {
     # Not allowed combinations: 
     # - platformName=python and debianFlavor=bookworm
     # - platformName=java and debianFlavor=bookworm
-    # - Any platformName other than dotnet and node js with debianFlavor=bookworm
-    if [ "$platformName" != "dotnet" ] && [ "$platformName" != "nodejs" ] && [ "$debianFlavor" == "bookworm" ]; then
+    # - Any platformName other than dotnet, node js, python, php and php-composer with debianFlavor=bookworm
```
[the remark this answers is not in the document]
Comment (kumaraksh1) [reply vsts/scripts/publishSdksFromStagingToProdStorageAccount.sh]: copyPlatformBlobsToProd function is called for all platforms then from this function copyPlatformBlobsToProdForDebianFlavor is being called for each debian flavor, so this condition is required

[the remark this answers is not in the document]
Comment (pauld-msft) [reply vsts/scripts/publishSdksFromStagingToProdStorageAccount.sh]: i guess my point was that i think that this check introduces some complexity that might not be needed. 

The platforms directory I linked above is what we use to determine what binaries to generate. So for [dotnet](https://github.com/microsoft/Oryx/tree/main/platforms/dotnet/versions), we create bookworm versions, push them to staging, and then copy them to prod with this script.

The platforms that aren't on the list you have here (ruby and golang) don't even _have_ bookworm versions, so they won't be generated / pushed to the staging repo. 

The only benefit that I would see for this check is if we want to generate binaries for those languages that get pushed to our staging account, but NOT to our prod account. I don't know that is a use case as of today that we care about

**Review by pauld-msft — APPROVED** [review]
discussed offline, and this check at least serves some purpose to help us control what versions are moved from staging to prod

**Pull request merged**